mito-ai/src/Extensions/InlineCompleter/provider.ts

```diff
@@ -414,18 +348,11 @@ interactive outputs.`
   }
 
   private _cleanCompletion(rawCompletion: string) {
-    let cleanedCompletion = rawCompletion;
-    if (this._currentPrefix) {
-      if (
-        cleanedCompletion.startsWith(this._currentPrefix) ||
-        this._currentPrefix.startsWith(cleanedCompletion)
-      ) {
-        cleanedCompletion = cleanedCompletion.slice(this._currentPrefix.length);
-      } else if (!['\n', ' '].includes(cleanedCompletion[0])) {
-        cleanedCompletion = '\n' + cleanedCompletion;
-      }
-    }
-    return cleanedCompletion;
+    return rawCompletion
```

Comment:
Can we double check that it's handling new line characters at the start of the completion correctly. 

Maybe the AI is just not returning a new line character here, or maybe we are ignoring it. 
<img width="330" alt="Screenshot 2024-12-27 at 11 50 30 AM" src="https://github.com/user-attachments/assets/90dd4751-1a4a-4caa-ac72-3410bf5b7910" />

Reply:
In some cases, it does appear to be working though 
<img width="513" alt="Screenshot 2024-12-27 at 11 53 14 AM" src="https://github.com/user-attachments/assets/5269f1b6-5ff5-40c4-a1e8-ced546b689e3" />

Reply:
Fixed this in-prompt. Tried to squeeze it into an existing instruction:

> 1. If the cursor appears at the end of a complete line (especially after a comment), ALWAYS start your code with a newline character

However, it just would not take. So now there's a specific line on formatting functions:

> 2. If the cursor appears at the end of a function definition, ALWAYS start your code with a newline character